frontend/app/login/page.tsx

```diff
@@ -28,10 +28,11 @@ const LoginInternal = () => {
     };
 
     return (
-        <div className="login-container">
-            <form className="login-form" onSubmit={handleSubmit}>
+        <div className={styles.loginBody}>
+            <div className={styles.loginContainer}>
+                <form className={styles.loginForm} onSubmit={handleSubmit}>
                 <h2>Login</h2>
-                <div className="form-group">
+                <div className={styles.formGroup}>
```

Comment:
why did these all get changed? did you change the framework you're using?

Reply:
I switched to using css modules for this dir, to isolate the CSS since it was leaking into some other stuff. This is the syntax we were using in other areas with those modules, so I decided to keep it consistent.